dangerfile.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
+"use strict";
+
+var fs   = require("fs"),
+	path = require("path"),
+
+	locater  = require("locater"),
+	pinpoint = require("pinpoint"),
+	dedent  = require("dedent"),
+
+	// Various views of changed/added files
+	jsfiles = danger.git.created_files
+		.concat(danger.git.modified_files)
+		.filter((file) => path.extname(file) === ".js"),
+
+	changelog = danger.git.modified_files.find((file) =>
+		file === "docs/change-log.md"
+	),
+
+	appfiles = jsfiles.filter((file) =>
+		file.indexOf("tests/") === -1
+	);
+
+function link(file, anchor, text) {
+	var repo = danger.github.pr.head.repo.html_url,
+		ref = danger.github.pr.head.ref;
+
+	return danger.utils.href(`${repo}/blob/${ref}/${file}${anchor || ""}`, file || text);
+}
+
+// All PRs should be targeted against `next`
+if(danger.github.pr.base.ref !== "next") {
+	warn("PRs should be based on `next`, rebase before submitting please");
+}
+
+// Any non-test JS changes should probably have a change-log entry
+if(appfiles.length && !changelog) {
+	warn(`Please include a ${link("docs/change-log.md", "changelog")} entry.`)
+}
+
+// Call out if `o.only(...)` was left in
+jsfiles
+	.filter((file) => file.indexOf("tests/") > -1)
```

Comment:
The arrow fn body? I've gone back and forth and settled on bracing only when necessary. Mithril doesn't have a set standard because it doesn't normally use arrow fns, but if we want to come to agreement I'm fine w/ following it.